app/api/routes/chat.py

```diff
@@ -56,8 +56,23 @@ def query(chat_request: ChatRequest):
 
 @router.post("/learn", status_code=204)
 def learn(learn_request: LearnRequest):
-    learn_collection.add(
-        documents=[learn_request.text],
-        ids=[uuid.uuid4().hex],
-    )
+
+    documents_paragraphs = learn_request.text.split("\n")
+
+    for document_sequence in documents_paragraphs:
+
+        split_into = 4
```

Comment:
I suggest moving `split_into` out of the loop since it doesn't need to be recreated on each iteration. Instead, define it as a constant (e.g., `SPLIT_INTO`) outside the loop.
Constants should be written in uppercase as per [PEP8](https://peps.python.org/pep-0008/#constants). Additionally, consider adding a comment explaining why we chose the specific value `4`. This provides context for collaborators and makes the code more maintainable.

Reply:
Thank you for your quick review :)
I will fix them!